src/app/app.component.html

```diff
@@ -386,22 +396,30 @@ <h1 *ngIf="!fatalMessageDetail" class="u-text-body u-text-center">
 <markdown-hints-dialog #markdownHintsDialog></markdown-hints-dialog>
 
 <!-- Footer -->
-<footer *ngIf="!curtain" class="o-container" id="footer" style="background-color: var(--color-lightgreen); margin-top: 30px;">
+<footer
+	*ngIf="!curtain"
+	class="o-container"
+	id="footer"
+	style="background-color: var(--color-lightgreen); margin-top: 30px"
+>
 	<div class="l-containerxaxis">
 		<div class="footer l-stickyfooter-x-footer">
-			<ul class="footer-x-leftlinks" style="width: 40%;">
-				<li class="Text-SemiBold" *ngIf="theme.showPoweredByOSL && theme.providedBy" [innerHtml]="'Start.poweredBy' | translate">
-				</li>
-			</ul>
+			<div class="tw-text-left tw-flex md:tw-flex-row md:tw-justify-between md:tw-gap-0 tw-flex-col-reverse tw-gap-[2em]">
+				<div>
+						<p
+							class="Text-SemiBold md:tw-w-1/2"
+							*ngIf="theme.showPoweredByOSL && theme.providedBy"
+							[innerHtml]="'Start.poweredBy' | translate"
+						></p>
+				</div>
 
-			<ul class="footer-x-links footer-rightside">
-				<li>
+				<div class="tw-flex tw-flex-col tw-gap-[1em] tw-min-w-fit">
 					<a class="faq" [routerLink]="['/public/faq']">FAQ</a>
-				</li>
-				<li>
-					<a class="impressum" [href]="theme.termsOfServiceLink" target="_blank">{{ 'Start.imprint' | translate }}</a>
-				</li>
-			</ul>
+					<a class="impressum" [href]="theme.termsOfServiceLink" target="_blank">{{
```

Comment:
Is the impressum link supposed to open a new tab?

Reply:
I just followed the previous impl. but I think we can make an update.